markdown/usage/introduction.md

```diff
@@ -24,86 +24,58 @@ The benefits of having your pipeline in Nextflow include:
+1. First, make sure that you have all required software installed (Nextflow + Docker / Singularity / Conda). See the [installation docs](installation.md) for more information.
 
-[Nextflow has built-in support for Conda](https://www.nextflow.io/docs/latest/conda.html) that allows us to manage dependencies using Conda recipes and environment files. The nf-core pipelines make extensive use of the bioinformatics conda channel [Bioconda](https://bioconda.github.io/) making it simple to install all required tools.
+    * Try running the Nextflow "hello world" example to make sure that the tools are working properly:
 
-Conda makes the installation and management of software dependencies far simpler than manual installations. It ensures correct version pinning and so represents good reproducibility.
-However, the software still runs in your operating system environment. As such, if it's possible to use either docker or singularity, those options are preferred.
+        ```bash
+        nextflow run hello
+        ```
 
-## How to run a pipeline
-In order to run a Nextflow pipeline from nf-core on your local computer you need to install Nextflow and Docker on your computer.
+2. Choose a pipeline to run. See the available pipelines at [https://nf-co.re/pipelines](https://nf-co.re/pipelines). If you have [nf-core/tools](tools.md) installed, run `nf-core list`
 
-**System requirements:**
-* Java 7 or 8
-* _recommended:_ at least 8GB of RAM
-* _recommended:_ Docker engine 1.10.x (or higher)
+3. Configure Nextflow to run on your system.
 
+    * The simplest way to run is with `-profile docker` (or `singularity` /  `conda`) which will tell Nextflow to execute jobs locally using Docker to fulfil the software requirements.
 
+    * For more complex configuration of Nextflow for your system, please see the [_Nextflow configuration_](https://nf-co.re/usage/configuration) documentation.
 
-1. Install Nextflow
+4. To test that everything is working properly, try running the tests for your pipeline of interest in the terminal:
 
     ```bash
-    curl -s https://get.nextflow.io | bash
+    nextflow run nf-core/<pipeline_name> -profile test,docker
     ```
 
-2. Install nf-core tools
+    * Replace `<pipeline_name>` with the name of an nf-core pipeline.
 
-    ```bash
-    pip install nf-core
-    ```
+    * If you don't have Docker installed, replace `docker` in the command with either `singularity` or `conda`.
 
-3. You can check all the pipelines available by typing in your terminal
+    * There is no need to download anything first - nextflow will pull the code for you from the GitHub repository automatically and fetch the software requirements too.
 
-    ```bash
-    nf-core list
-    ```
+    * If the pipeline fails, check the [troubleshooting docs](/usage/troubleshooting.md) and ask for help on the nf-core Slack channel for that particular pipeline (see [https://nf-co.re/join](https://nf-co.re/join)).
```

Comment:
```suggestion
    * If the pipeline fails, check the [troubleshooting docs](troubleshooting.md) and ask for help on the nf-core Slack channel for that particular pipeline (see [https://nf-co.re/join](https://nf-co.re/join)).
```
I think we're already in `usage/` here in `introduction.md`, so the link should be `troubleshooting.md`

Reply:
True. Should work on the website, as the first `/` sets the path to the web root. But it'll probably break on GitHub.